pages/Front_page.md

```diff
@@ -69,4 +69,4 @@ Email: fair-universe@lbl.gov
 
 Updates will be announced through fair-universe-announcements google group. [Click to join Google Group](https://groups.google.com/u/0/a/lbl.gov/g/Fair-Universe-Announcements/)
 
-Join **#higgsml-uncertainty-challenge** channel in [FAIR Universe slack workspace](https://join.slack.com/t/fairuniverse/shared_invite/zt-2dt9ovrp1-jvi0DnCK9jzL3VGrdwYNMA)
\ No newline at end of file
+Join **#higgsml-uncertainty-challenge** channel in [FAIR Universe slack workspace](https://join.slack.com/t/fairuniverse/shared_invite/zt-2dt9ovrp1-jvi0DnCK9jzL3VGrdwYNMA)
```

Comment:
these 2 lines look the same. i don't know what is the difference

Reply:
I made this PR straight from GitHub so it adds a extra white space or something. either way not an issue